demo/index.html

```diff
@@ -257,101 +245,18 @@ <h3><code>iron-form</code> respects nested elements</h3>
 
             <input type="submit" value="Submit">
             <br>
-            <paper-button raised="" onclick="form6.submit()">Submit</paper-button>
-            <paper-button raised="" onclick="form6.reset()">Reset</paper-button>
+            <paper-button raised onclick="form6.submit()">Submit</paper-button>
+            <paper-button raised onclick="form6.reset()">Reset</paper-button>
           </form>
           <div class="output"></div>
         </iron-form>
         <script>
           form6.addEventListener('iron-form-submit', function(event) {
             this.querySelector('.output').innerHTML = JSON.stringify(event.detail);
           });
-        &lt;/script>
-      </template>
-    </demo-snippet>
-  </div>`;
-
-document.body.appendChild($_documentContainer.content);
```

Comment:
2 demos were duplicated, don't know why. this entire file was modulized super weirdly